### ISSUE TITLE
Would displaying suggested request ideas on Alaveteli sites encourage more people to make requests?

### ISSUE THREAD

**Issue by Gemmamysoc** [mysociety/alaveteli-experiments#106]
## Q: Would having a 'Need inspiration?' tab/page on Alaveteli sites encourage more people to make requests?

The list of suggested requests could be inspired by [this list](https://groups.google.com/forum/#!topic/alaveteli-users/0Hyk5tj3aHQ) or could be a specially-curated list of requests for information that currently isn't already public in that country. 

[Explain how you will measure this]

It could be measured by running an A/B test for a few months - one version of site would display the suggested requests page, and the other version wouldn't.  Then we would measure which version of the site had the most people making requests. 

Or, the suggested requests page could be added to one Alaveteli site (e.g. Australia) and we could measure if any more requests were made after the addition/after people clicked on it. 

[What do we need to do to be able to take the measurements?]

Access to Google Analytics

**Comment by andreaspavlou** [comment]
Sounds good. It could be interesting to also make these requests easy to submit with some kind of ´click here´ to automatically send the request? That could become a feature for doing campaigns or mass requests as well.

**Comment by Gemmamysoc** [comment]
Yeah, great idea Andreas. You can already generate links to pre-filled requests to certain bodies. Here's an example: https://www.righttoknow.org.au/new/nsw_police_force?title=Police%20brutality&default_letter=How%20many%20complaints%20of%20police%20brutality%20were%20received%20by%20your%20police%20force%20between%201st%20July%202015%20and%201st%20July%202016%3F%0A 

Here are instructions on how to do this: https://www.whatdotheyknow.com/help/api

I might actually write a blog post too about this feature, to promote it a bit more. Do you think that would be useful?

**Comment by andreaspavlou** [comment]
Yeh it would be super useful! :) Something step by step would be helpful to understand for us non techies :)

**Comment by Gemmamysoc** [comment]
So, another way of experimenting with this could be to add pre-filled request urls to the notes fields of public bodies who don't currently get many requests. The pre-filled requests would have to be for information/data that isn't currently available on the Alaveteli site or the gov department's own sites.

**Comment by Gemmamysoc** [comment]
Here is a draft blog post/documentation page about using the pre-filled request tool: https://docs.google.com/document/d/19s6Hhu94g1IJFBev3V-bSVEupsj0lOq0FrLcpVQOn5Y/edit# As a campaigner @andreaspavlou, I'd be interested to hear if this is understandable and useful for you! Cheers :)
